docs/generated/devkit/ProjectGraphExternalNode.md

```diff
@@ -14,8 +14,8 @@ while allowing tracking of the full tree of different nested versions
 ### Properties
 
 - [data](../../devkit/documents/ProjectGraphExternalNode#data): Object
-- [name](../../devkit/documents/ProjectGraphExternalNode#name): `npm:${string}`
-- [type](../../devkit/documents/ProjectGraphExternalNode#type): "npm"
+- [name](../../devkit/documents/ProjectGraphExternalNode#name): `npm:${string}` | `cargo:${string}` | `gradle:${string}`
```

Comment:
imho this should just be `string`, nx-dotnet would likely add a new one, nxlv/poetry would add a new one etc.